pythonx/cm_sources/alchemist.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# For debugging, use this command to start neovim:
+#
+# NVIM_PYTHON_LOG_FILE=nvim.log NVIM_PYTHON_LOG_LEVEL=INFO nvim
+
+import os, sys
+PLUGIN_BASE_PATH = os.path.abspath("%s/../../../" % __file__)
+sys.path.insert(0, PLUGIN_BASE_PATH)
+from elixir_sense import ElixirSenseClient
+import re
+
+DEBUG = False
+
+from cm import register_source, getLogger, Base
+register_source(name='alchemist',
```

Comment:
@hauleth I believe you can help here to do a better configuration

Reply:
I have ditched neovim-completion-manager in favour of asyncomplete recently, but I will try.